multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/bootstrap/BootstrapServlet.java

```diff
@@ -110,31 +102,4 @@ private void initializeProviders() {
         AuditLoggingProvider.setFacade(new AuditLoggingFacadeSLImpl(dataSource, getUserInfoProvider()));
     }
 
-    private void executeAsyncDatabaseChanges() {
```

Comment:
Now that this is gone, when will the changes be made?
Or do we even need to make these changes on app startup?